docs/user-guide/configuring.md

```diff
@@ -578,7 +578,7 @@ You can also use your `.gitignore` file:
 
     eslint --ignore-path .gitignore file.js
 
-Any file that follows the standard ignore file format can be used. Keep in mind that specifying `--ignore-path` means that any existing `.eslintignore` file will not be used.
+Any file that follows the standard ignore file format can be used. Keep in mind that specifying `--ignore-path` means that any existing `.eslintignore` file will not be used. Note that globbing rules at .eslintignore are more strict that these from .gitignore. See all supported patterns in [minimatch docs](https://github.com/isaacs/minimatch)
```

Comment:
"Note that globbing rules in `.eslintignore` are more strict than in `.gitignore`."